samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -625,6 +624,22 @@
                 }
               ]
             }
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "dynamodb:ListStreams"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${tableName}/stream/*",
```

Comment:
I have addressed your pull request comments.
Also I verified transformed template deploys and application functions as expected.

Ready for review again.